pkg/virtctl/portforward/tcp.go

```diff
@@ -4,10 +4,11 @@ import (
 	"io"
 	"net"
 
-	"github.com/golang/glog"
+	"kubevirt.io/client-go/log"
 )
 
 func (p *portForwarder) startForwardingTCP(address *net.IPAddr, port forwardedPort) error {
+	log.InitializeLogging("portforward")
```

Comment:
Is it necessary to initialize the logger multiple times?

Reply:
It shouldn't be needed. Have made changes to initialize only once per package in the entrypoint function in the package.